menas-web/Dockerfile

```diff
@@ -11,7 +11,10 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
-FROM nginx:1-alpine
+ARG DOCKER_BASE_ARTIFACT=nginx
+ARG DOCKER_BASE_TAG=1-alpine
+
+FROM ${DOCKER_BASE_ARTIFACT}:${DOCKER_BASE_TAG}
```

Comment:
I know this is a new feature that people are excited about, but I think it was mainly created for multi staged builds.

What does it give us here? What are the pros of this approach? I honestly see only possible problems here + the extra (undeclared) properties in Maven.

Reply:
I would put a comment in there above the ARGVs as to why it is like that.